format.md

```diff
@@ -99,19 +99,25 @@ scenario in YAML format.
     Algorithms are terminated if their memory exceeded this value. Might
     be used both for `runtime` and `solution_quality`. Put `?` if it was not set for experiment.
 -   `features_cutoff_time [integer]`: Cut-off time in seconds --
-    Feature set computation is terminated if it exceeds this time. Might
+    Instance feature set computation is terminated if it exceeds this time. Might
     be used both for `runtime` and `solution_quality`. Put `?` if it was not set for experiment.
 -   `features_cutoff_memory [integer]`: Cut-off memory in megabytes -- 
-    Feature set computation is terminated if memory exceeded this value.
+    Instance feature set computation is terminated if memory exceeded this value.
+    Might be used both for `runtime` and `solution_quality`. Put `?` if it was not set for experiment.
+-   `algorithm_features_cutoff_time [integer]`: Cut-off time in seconds --
+    Algorithmic feature set computation is terminated if it exceeds this time. Might
+    be used both for `runtime` and `solution_quality`. Put `?` if it was not set for experiment.
+-   `algorithm_features_cutoff_memory [integer]`: Cut-off memory in megabytes -- 
+    Algorithmic feature set computation is terminated if memory exceeded this value.
     Might be used both for `runtime` and `solution_quality`. Put `?` if it was not set for experiment.
 -   `number_of_feature_steps [integer]`: Number of feature steps.
 -   `feature_steps [list of objects describing feature steps]:`
     Objects specifying the feature step name, the features it provides,
     and any dependencies on other feature steps. For instance, probing
     features need normally a preprocessing step. Therefore probing
     features depend on the preprocessing step and the probing step. This
-    is helpful for `feature_costs.arff`, as feature generators often calculate features in
-    several steps. Each feature has to be listed in at least one step.
+    is helpful for `feature_costs.arff` or `algorithm_feature_costs.arff`, as feature generators 
```

Comment:
that's the only place where we mix instance features and algorithm features into one list. I would recommend to make this consistent by splitting this into `feature_steps` and `algorithm_feature_steps `

Reply:
I added a separate field for `algorithm_feature_steps`. 

Thanks!